src/main/java/org/team1540/base/ChickenSubsystem.java

```diff
@@ -11,14 +23,57 @@ public ChickenSubsystem(String name) {
   public ChickenSubsystem() {
   }
 
-  protected abstract double getCurrent();
+  protected void initDefaultCommand() {
+
+  }
 
   /**
-   * Set a power limit for this subsystem.
+   * Compare two ChickenCommands by priority.
+   *
+   * @param o ChickenCommand to compare to.
    *
-   * @param limit The power limit in amps.
+   * @return priority.compareTo(o.priority)
    */
-  protected abstract void limitPower(double limit);
+  public int compareTo(ChickenSubsystem o) {
+    return Double.compare((priority), o.priority);
+  }
+
+  public double getPriority() {
+    return priority;
+  }
+
+  public void setPriority(double priority) {
+    this.priority = priority;
+  }
 
-  protected abstract void stopLimitingPower();
+  public double getCurrent() {
+    double sum = 0;
+    for (CANTalon currentMotor : motors) {
+      sum += currentMotor.getOutputCurrent();
+    }
+    return sum;
+  }
+
+  public void limitPower(double limit) {
+    for (CANTalon currentMotor : motors) {
+      currentMotor.EnableCurrentLimit(true);
+      currentMotor.setCurrentLimit(Math.toIntExact(Math.round(limit)));
+    }
+  }
+
+  public void stopLimitingPower() {
+    for (CANTalon currentMotor : motors) {
+      currentMotor.EnableCurrentLimit(false);
+    }
+  }
+
+  /**
+   * Compare two PowerManageables by priority.
+   *
+   * @param o PowerManageables to compare to.
+   * @return priority.compareTo(o.priority)
+   */
+  public int compareTo(PowerManageable o) {
```

Comment:
For cleanliness' sake, maybe just do your own thing (o.priority - priority or maybe it's the other way around) instead of calling a different function?